aws-rds-customdbengineversion/inputs/inputs_1_create.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Engine": "custom-oracle-ee-cdb",
+  "DatabaseInstallationFilesS3BucketName": "{{S3BucketNamePrefix}}",
+  "DatabaseInstallationFilesS3Prefix": "2020-10",
+  "KMSKeyId": "{{KMSKeyArn}}",
```

Comment:
Same idea: we need to have exported  KmsKeyArn resouce via dependencies stack

Reply:
Dependencies stack is provided in deploying stage.